app.js

```diff
@@ -509,11 +514,24 @@ document.querySelectorAll('.double-slider').forEach(parent => {
 
 		if (e.target.className == 'va') FILTER.school[0] = e.target.value;
 		else FILTER.school[1] = e.target.value;
-		
+		localStorage.setItem('filter', JSON.stringify(FILTER));
+
 		clearTimeout(slider_timer);
 		slider_timer = setTimeout(() => {
 			render();
 			CALENDAR.refresh();
 		}, 300);
 	}, false);
 });
+
+document.querySelectorAll('.va').forEach( el => {
```

Comment:
do these lines have something to do with persistent filters?

Reply:
yes, set double slider

Reply:
seems more like a double slider than persistent filter thing to me :D